src/test/util_tests.cpp

```diff
@@ -132,6 +132,27 @@ BOOST_AUTO_TEST_CASE(util_FormatISO8601Date)
     BOOST_CHECK_EQUAL(FormatISO8601Date(1317425777), "2011-09-30");
 }
 
+BOOST_AUTO_TEST_CASE(util_time_GetTime)
+{
+    SetMockTime(111);
+    // Check that mock time does not change after a sleep
+    for (const auto& num_sleep : {0, 1}) {
```

Comment:
Why the loop? Isn't enough one loop after `MilliSleep(1)`?

Reply:
Yeah, those are unit tests. Probably enough to not have them at all :woman_shrugging: